src/main/java/com/aws/greengrass/deployment/converter/DeploymentDocumentConverter.java

```diff
@@ -201,6 +205,36 @@ private static String parseGroupNameFromConfigurationArn(Configuration config) {
         return groupName;
     }
 
+    @SuppressWarnings("PMD.PreserveStackTrace")
+    private static void convertThingGroupArns(DeploymentDocument.DeploymentDocumentBuilder deployDocBuilder,
```

Comment:
I'm on mobile right now so it is a bit hard to see. What are we doing when both of the new fields are not present, which would be the case for all deployments today?

Reply:
Nothing! If they are empty, we don't do arn parsing and the deployment would work as is.